ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/security-token-service/deprovision/cucushift-installer-rehearse-aws-ipi-security-token-service-deprovision-chain.yaml

```
@@ -0,0 +1,11 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-security-token-service-deprovision
```

```suggestion
  as: cucushift-installer-rehearse-aws-ipi-cco-manual-security-token-service-deprovision
```

To keep consistent with the naming of other sts install workflow.